src/modules/flight_mode_manager/FlightModeManager.cpp

```diff
@@ -415,30 +429,79 @@ void FlightModeManager::send_vehicle_cmd_do(uint8_t nav_state)
 	}
 
 	// publish the vehicle command
+	command.timestamp = hrt_absolute_time();
 	_vehicle_command_pub.publish(command);
 }
 
 void FlightModeManager::generateTrajectorySetpoint(const float dt,
 		const vehicle_local_position_s &vehicle_local_position)
 {
+	// In case flight task was not able to update correctly we send the empty setpoint which makes the position controller failsafe.
```

Comment:
Thinking about it more in detail there is a difference in conditioning and that's why this part should be in a separate `handleCommand` function that gets called just after `start_flight_task();` not conditioned with `isAnyTaskActive()`.

Otherwise there has to be already a task active before you can command another one. In reality this means you cannot issue and Orbit when you're currently flying e.g. Stabilized. Not a big impact. I can fix it on top.

Reply:
Fixed in 2c3286d